.env

```diff
@@ -0,0 +1 @@
+REACT_APP_MAP_API=pk.eyJ1IjoiamtpZXJuYW4xMiIsImEiOiJja29pdWFkb2cxNzV2Mm9taTQzcG9uMjBlIn0.Hz5wD1J85TPPuYOHHtkUxQ
```

Comment:
Can we add .env to our .gitignore file so the key doesn't get pushed to production?